src/vuex/modules/ledger.js

```diff
@@ -9,6 +9,27 @@ export default () => {
   const mutations = {}
 
   const actions = {
+    async showAddressOnLedger() {
+      const { default: Ledger } = await import("@lunie/cosmos-ledger")
+
+      const ledger = new Ledger({
+        testModeAllowed: state.externals.config.testModeAllowed
+      })
+
+      try {
+        await ledger.confirmLedgerAddress()
```

Comment:
what happens if you click this button but the ledger is not plugged in? or on screensaver mode? or not in Cosmos app?

Reply:
🤷‍♂

Reply:
![image](https://user-images.githubusercontent.com/5869273/71937333-1b2e1f00-317a-11ea-9c1f-be500f79645f.png)